packages/overlays/src/ArrowMixin.js

```diff
@@ -24,7 +24,7 @@ export const ArrowMixinImplementation = superclass =>
     }
 
     static get styles() {
-      const superCtor = /** @type {typeof import('@lion/core').LitElement} */ (super.constructor);
+      const superCtor = /** @type {typeof import('@lion/core').LitElement} */ (this.constructor);
```

Comment:
```suggestion
      const ctor = /** @type {typeof import('@lion/core').LitElement} */ (this.constructor);
```

Reply:
Hmm, this is not the right fix at all. Thanks for noticing, it should have been just `super`, because you are already in 'constructor context'. Still waking up atm :)